ansible/roles/iroha-docker/README.md

```diff
@@ -156,3 +156,61 @@ Deploying 6 Iroha peers on two remote hosts communicating over overlay network (
 
 ##### Caveats
 1. If `/usr/bin/python` does not exist on a remote host, Ansible will fail with the misleading message: `... Make sure this host can be reached over ssh`. This usually happens when Ansible uses Python 3. On Ubuntu systems `/usr/bin/python3` is not symlinked to `/usr/bin/python` which Ansible expects to find. The problem can be solved by setting `ansible_python_interpreter` variable to `/usr/bin/python3`.
+
+
+# Migration of WSV from Postgres to Rocksdb
+
+Migraion from Postgres to Rocksdb replaces [WSV](https://iroha.readthedocs.io/en/develop/concepts_architecture/architecture.html) from Postgres to Rocksdb.
+
+During the migration blockstore stays the same, postgres container is stopped and rocksdb is located inside iroha containers.
+
+
+# Migration process
+**Make sure that you read the documentation before migration**
+
+Migration is possible only in one direction. If you migrate WSV to Rocksdb and update blockchain, you will not be able to migrate new blocks to Postgres WSV,this case is not considered by developers. But it is possible to restore the state of Postgres WSV (the state right before migration), if Postgres container is not deleted (migration only stops the container). So do not delete Postgres unless you are sure that everything works correctly after migration.
+
+## Prerequisites before migration: Usual run parameters with Postgres WSV
+Before the migration, check that Postgres was runned with these parameters in ansible defaults. Also, Postgres container should be running.
+
+```
+iroha_use_rdb: False
+iroha_migrate_rdb: False
```

Comment:
I think it would be better to rename the variables to `_rocksdb*` instead of `_rdb*` because rdb usually refers to Relation DB and it could be confusing for the uninitiated user.